apps/microtvm/zephyr/template_project/microtvm_api_server.py

```diff
@@ -195,6 +195,30 @@ def _get_device_args(options):
     )
 
 
+def _get_board_mem_size(options):
+    board_file_path = (
+        pathlib.Path(get_zephyr_base(options))
+        / "boards"
+        / "arm"
+        / options["zephyr_board"]
+        / (options["zephyr_board"] + ".yaml")
+    )
+    try:
+        with open(board_file_path) as f:
+            board_data = yaml.load(f, Loader=yaml.FullLoader)
+            return int(board_data["ram"]) * 1024
+    except:
+        _LOG.warning("Board memory information is not available.")
+    return None
+
+
+def _get_recommended_heap_size(options):
+    prop = BOARD_PROPERTIES[options["zephyr_board"]]
+    if "recommended_heap_size" in prop:
+        return prop["recommended_heap_size"]
+    return 216 * 1024
```

Comment:
change this value to a global constant with capital letters for better readability.